### DISCUSSION TITLE
how to enable overlay scrollbar on windows 10?

### DISCUSSION THREAD

**Discussion by TTTPOB** [tauri-apps/tauri#5988] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19909103/210770985-e88f689b-1a3d-4a8f-b6c7-5338ad74cc66.png)
first day on tauri.
I created a toy project and saw the scrollbar, in edge we can already enable overlay scrollbar via `edge://flags` or `--enable-features` commandline flag, but how can I do this in tauri? 
is there a way to pass options to windows webview2 like [this link](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2304)

**Comment by amrbashir** [comment, marked as answer]
the option to pass browser arguments will be available in the next release.

**Comment by gihanrangana** [comment]
> the option to pass browser arguments will be available in the next release.

is this available now?

**Reply by FabianLars** [reply]
Yes, it was added in v1.3

You can set the args in [tauri.conf.json](https://tauri.app/v1/api/config/#windowconfig.additionalbrowserargs) and [rust](https://docs.rs/tauri/latest/tauri/window/struct.WindowBuilder.html#method.additional_browser_args).

**Reply by M4n5ter** [reply]
It doesn't seem to work in my environment, I don't know if it's a problem with webview2.
https://github.com/MicrosoftEdge/WebView2Feedback/issues/2304#issuecomment-1248689881
```json
"windows": [
            {
                "decorations": false,
                "fullscreen": false,
                "height": 600,
                "minHeight": 550,
                "minWidth": 350,
                "title": "Tauri-quick-start",
                "width": 800,
                "additionalBrowserArgs": "--disable-features=msWebOOUI,msPdfOOUI,msSmartScreenProtection --enable-features=OverlayScrollbar,msOverlayScrollbarWinStyle,msOverlayScrollbarWinStyleAnimation"
            }
        ],
```
![image](https://github.com/tauri-apps/tauri/assets/68144809/4116d66a-1670-4860-98e5-0878b0a52200)

```bash
cargo tauri info                                                                                
                                                                                                                   
[✔] Environment                                                                                                    
    - OS: Windows 10.0.22631 X64
    ✔ WebView2: 121.0.2277.112
    ✔ MSVC: Visual Studio ���ɹ��� 2022
    ✔ rustc: 1.78.0-nightly (8ace7ea1f 2024-02-07)
    ✔ cargo: 1.78.0-nightly (ccc84ccec 2024-02-07)
    ✔ rustup: 1.26.0 (5af9b9484 2023-04-05)
    ✔ Rust toolchain: nightly-x86_64-pc-windows-msvc (environment override by RUSTUP_TOOLCHAIN)
    - node: 20.11.0
    - npm: 10.2.4

[-] Packages
    - tauri [RUST]: 2.0.0-beta.2
    - tauri-build [RUST]: 2.0.0-beta.1
    - wry [RUST]: 0.35.2
    - tao [RUST]: 0.25.0
    - tauri-cli [RUST]: 2.0.0-beta.1
    - @tauri-apps/api [NPM]: 1.5.3
    - @tauri-apps/cli [NPM]: 2.0.0-beta.1

[-] App
    - build-type: bundle
    - CSP: default-src blob: data: filesystem: ws: wss: http: https: tauri: 'wasm-unsafe-eval' 'self'; style-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-inline' 'wasm-unsafe-eval' 'self'; script-src blob: data: filesystem: ws: wss: http: https: tauri: 'wasm-unsafe-eval' 'self'; connect-src ipc: http://ipc.localhost        
    - frontendDist: ../frontend/dist
    - devUrl: http://127.0.0.1:42069/
```

**Reply by Livven** [reply]
In Tauri 2 it seems like the `--disable-features` argument is causing issues, so if you remove it i.e. pass only `--enable-features=OverlayScrollbar` it works (`msOverlayScrollbarWinStyle` etc. is also unnecessary).
